tests/commits/check.sh

```diff
@@ -40,7 +40,7 @@ git log --no-merges --pretty="%H" $commit_range -- | (
 
     subject=$(git show -s --format='%s' $commit)
 
-    if ! echo "$subject" | egrep -q '^[a-z/_-]+:'; then
+    if ! echo "$subject" | egrep -q '^[a-zA-Z0-9/(){}$,._-]+:'; then
```

Comment:
I was kinda happy with the strict version, especially not to include upper case letters :) I see the value in the rest modification. As this is a very opinionated topic, it is not a change request.